websrc/source/documents/tutorials/contiv-policy-kubernetes-16.md

```diff
@@ -117,9 +120,15 @@ Let's make sure our pods are up and running.
 
 ```
 [vagrant@kubeadm-master ~]$ kubectl get pods -o wide
-NAME      READY     STATUS    RESTARTS   AGE       IP         NODE
-apod      1/1       Running   0          1m        10.1.1.1   kubeadm-worker0
-bpod      1/1       Running   0          1m        10.1.1.2   kubeadm-worker0
+NAME                        READY     STATUS    RESTARTS   AGE       IP         NODE
+apod                        1/1       Running   0          35s       10.1.1.1   kubeadm-worker0
+bpod                        1/1       Running   0          19s       10.1.1.2   kubeadm-worker0
+contiv-blue-c1              1/1       Running   0          15m       10.1.2.2   kubeadm-worker0
+contiv-blue-c2              1/1       Running   0          15m       10.1.2.3   kubeadm-master
+contiv-blue-c3              1/1       Running   0          15m       10.1.2.4   kubeadm-worker0
+contiv-c1                   1/1       Running   0          19m       10.1.2.2   kubeadm-master
+contiv-c2                   1/1       Running   0          19m       10.1.2.3   kubeadm-worker0
+vanilla-c-357166617-s5tff   1/1       Running   1          20m       20.1.1.5   kubeadm-worker0
 ```
 Now try to ping from apod to bpod. They should be able to ping each other.
```

Comment:
>>Docker legacy swarm tutorial 
I see , Could you change it,  so that it is easier to understand for the first-timer ?